contrib/debian/rules

```diff
@@ -6,7 +6,7 @@
 #	$(if $(filter nocheck,$(DEB_BUILD_OPTIONS)),,src/test_bitcoin)
 
 DEB_INSTALL_EXAMPLES_bitcoind += debian/examples/*
-DEB_INSTALL_MANPAGES_bitcoind += debian/manpages/*
+DEB_INSTALL_MANPAGES_bitcoind += doc/man/*
```

Comment:
I think so too - our installer installs the man pages, there's no need for debian to do anything special.